ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/css/formcontainerview.css

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  ******************************************************************************/
 
-[data-cmp-visible="false"] {
+html:not(.aem-AuthorLayer-Edit) [data-cmp-visible="false"] {
```

Comment:
site's css should anyways not be loaded in editor. isn't it the case already?

Reply:
This css is loaded as it is part of the runtime.

Reply:
we didn;t use to load any runtime lib (except for base I think) in the editor. has that changed?

Reply:
We don't load runtime lib in editor.

Reply:
On further analysis: we actually load runtime css in the editor to have the look and feel same for WYSIWYG authoring.